src/universal/modules/teamDashboard/components/ArchiveTeam/ArchiveTeamForm.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import React, {Component} from 'react';
+import {reduxForm, Field} from 'redux-form';
+import InputField from 'universal/components/InputField/InputField';
+
+type Props = {
+  handleFormBlur: () => any,
+  handleFormSubmit: () => any,
+  handleSubmit: (fn: () => any) => any,
+  teamName: string
+};
+
+class ArchiveTeamForm extends Component<Props> {
+  validate = (value) => (
+    value !== this.props.teamName
+      ? 'The team name entered was incorrect'
+      : undefined
+  );
+
+  render() {
+    const {handleFormSubmit, handleFormBlur, handleSubmit} = this.props;
+    return (
+      <form onSubmit={handleSubmit(handleFormSubmit)}>
+        <Field
```

Comment:
Just curious, do input fields get minimal a11y love, like an `aria-label`? Not in the scope of this PR, for sure. I could look it up, too. Ha! #lazyweb